usefulUtilities/botPlayer/README.md

```diff
@@ -0,0 +1,14 @@
+# Bot Player
+Bot Player helps with domain testing bots that need to be placed in a certain location, are a certain unique set, and that need to last through domain resets.
+
+## Setup
+1. use the botRecorder.js to record the bots that you need.  When you click on the tablet button, it will start recording and then stop after you click it again. 
+2. Keep the script update up to check on the status and to see where the file was saved.
+3. Upload your bot files to a s3 directory(this is currently hardcoded in the manager file)
+4. Add the following two scripts to the Domain scripts [Place here when ready].
```

Comment:
what's "place here when ready" ?

Reply:
that's for me to update when I have actual release files.  I can make them now though.

Reply:
done